pkgs/by-name/up/updatecli/package.nix

```
@@ -52,9 +61,10 @@ buildGoModule rec {
       Updatecli is a command-line tool used to define and apply update strategies.
     '';
     homepage = "https://www.updatecli.io";
-    changelog = "https://github.com/updatecli/updatecli/releases/tag/v${version}";
+    changelog = "https://github.com/updatecli/updatecli/releases/tag/${src.rev}";
     license = licenses.asl20;
     mainProgram = "updatecli";
+    platforms = platforms.linux;
```

This seems to support macos as well

```suggestion
    platforms = platforms.unix;
```

---

If it supports macos, we don't need platforms than. Because buildGoPackage already sets a platforms field by default.

---

default platforms include wasi, not sure if that's applicable here.

---

I don't see anyone explicitly excluding wasi anywhere. Since it's not an official platforms does it makes sense to exclude it? It might be more work in future to remove all the platforms restriction if someone actually want to support wasi. i.e. the "unix" platform will also not allow builds on windows in future, which we might support to some basic extend in 1 or 2 years: https://github.com/NixOS/nix/pulls?q=is%3Apr+is%3Aopen+windows

---

Thank you both for the input! 
I wasn't aware that `buildGoModule` (and most build helpers for other languages as well) already provide sensible defaults for `platforms`.
For this package I agree that the defaults are fine, so I removed this change!